front/lib/api/assistant/global_agents/configurations/dust/dust.ts

```diff
@@ -129,22 +134,29 @@ ALWAYS provide a \`limit\` when using \`cat\`. The maximum \`limit\` is 10,000 c
 </cat_tool_guidelines>
 </company_data_guidelines>`,
 
-  warehouses: `<data_warehouses_guidelines>
-You can use the Data Warehouses tools to:
-- explore what tables are available in the user's data warehouses
-- describe the tables structure
-- execute a SQL query against a set of tables
-
-In order to properly use the data warehouses, it is useful to also search through company data in case there is some documentation available about the tables, some additional semantic layer, or some code that may define how the tables are built in the first place.
-Tables are identified by ids in the format 'table-<dataSourceId>-<nodeId>'.
-The dataSourceId can typically be found by exploring the warehouse, each warehouse is identified by an id in the format 'warehouse-<dataSourceId>'.
-</data_warehouses_guidelines>`,
+  toolsets: (availableToolsets: MCPServerViewResource[]) => {
```

Comment:
Should we also do it for deep dive ?

Reply:
BTW the deep dive prompt around toolsets should also evolve (but probably off topic for this PR)